src/common/transformations/src/transformations/common_optimizations/convert_quantize_dequantize.cpp

```diff
@@ -135,6 +138,14 @@ ov::pass::ConvertQuantizeDequantize::ConvertQuantizeDequantize() {
             if (out_low_val != 0 || out_high_val != 255)
                 return false;
             break;
+        case element::Type_t::i16:
+            if (out_low_val != -32768 || out_high_val != 32767)
+                return false;
+            break;
+        case element::Type_t::u16:
+            if (out_low_val != 0 || out_high_val != 65536)
```

Comment:
After fixing in QuantizeLinear decomposition, it should be reverted back to 65535.